x-pack/plugins/apm/server/lib/helpers/transaction_group_query.ts

```diff
@@ -5,20 +5,13 @@
  */
 
 import moment from 'moment';
-import { oc } from 'ts-optchain';
 import {
   TRANSACTION_DURATION,
   TRANSACTION_NAME
 } from '../../../common/constants';
-import { Transaction, TransactionV2 } from '../../../typings/Transaction';
+import { Transaction } from '../../../typings/Transaction';
 import { ITransactionGroup } from '../../../typings/TransactionGroup';
 
-interface ITransactionGroupSample {
-  transaction: Transaction['transaction'];
-  context: Transaction['context'];
-  trace?: TransactionV2['trace'];
-}
-
```

Comment:
Again: nice! 
Striking the right balance between having accurate interfaces, but the fewer the better, is hard.